tokio/src/runtime/tests/loom_join_set.rs

```diff
@@ -1,6 +1,8 @@
 use crate::runtime::Builder;
+#[cfg(tokio_unstable)]
 use crate::task::JoinSet;
```

Comment:
I believe you should be able to just put this on the entire file like this:
```suggestion
#![cfg(tokio_unstable)]
use crate::runtime::Builder;
use crate::task::JoinSet;
```